### PR TITLE
TN-3222 Feature/2fa remember me

### PR DESCRIPTION
**Pull request by achen2401**
part of https://jira.movember.com/browse/TN-3222
Frontend changes:
- add a remember me checkbox on the 2FA verify page (see screenshot):
Screen Shot 2023-05-23 at 1 50 13 PM
- when the checkbox is checked a cookie will be stored in the browser that will expire in 14 days (see screenshot) (backend will need to store this value for comparison later?):
Screen Shot 2023-05-23 at 2 19 52 PM
- this cookie value will be sent to the backend upon login form submission and the backend will use it to determine whether to present 2FA verify page to the user. (see screenshot of input value submitted to the backend)
Screen Shot 2023-05-23 at 1 46 04 PM

### CODE REVIEW

**Review by pbugni — COMMENTED** [review]
@achen2401 this looks great.  Not approving as I will use the same branch/PR to implement the back end portions, and expect we may have more back and forth before this feature is complete.

File: portal/static/js/flask_user/2fa.js [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+  $("#" + RESENT_CODE_BUTTON_ID).on("click", function (e) {
+    e.stopPropagation();
+    submitForm(e.target);
+  });
+  $("#" + VALIDATE_CODE_BUTTON_ID).on("click", function (e) {
+    e.stopPropagation();
+    submitForm(e.target);
+  });
+  $(document).keypress(function (event) {
+    if (event.which === 13) {
```
[the remark this answers is not in the document]
Comment (achen2401) [reply portal/static/js/flask_user/2fa.js:33]: @pbugni `13` == key code for the `Enter` key.

**Review by pbugni — APPROVED** [review]

**Comment by mcjustin** [comment]
I played w/ this on the test system, the behavior meets my expectations. Didn't do a full QA, though I did try both checking the box and not. 
Thanks! 

@achen2401  one tiny thing that I'd like corrected, but which was present prior to this change. Can you please capitalize the "l" in "log in"? I think we use that casing consistently.
image

**Comment by achen2401** [comment]
> @achen2401 one tiny thing that I'd like corrected, but which was present prior to this change. Can you please capitalize the "l" in "log in"? I think we use that casing consistently.

@mcjustin This is fixed.  Thank you for flagging that.
screenshot after fix:
Screen Shot 2023-05-31 at 10 29 49 AM

**Review by mcjustin — APPROVED** [review]

**Pull request merged**